docs/new-command.md

```diff
@@ -971,7 +971,76 @@ Failure to call `base.Dispose()` will prevent request and response data from `Ca
    - Include examples in description
    - **Maintain alphabetical sorting in e2eTestPrompts.md**: Insert new test prompts in correct alphabetical position by Tool Name within each service section
 
-5. Live Test Infrastructure:
+5. **Tool Description Quality Validation**:
+   - Test your command descriptions for quality using the validation tool before submitting:
+
+    - **Single prompt validation** (test one description against one prompt):
+
+     ```bash
+     cd eng/tools/PromptConfidenceScore
+     dotnet run -- --validate --tool-description "Your command description here" --prompt "typical user request"
+     ```
+
+   - **Multiple prompt validation** (test one description against multiple prompts):
+
+     ```bash
+     dotnet run -- --validate \
+       --tool-description "Lists all storage accounts in a subscription" \
+       --prompt "show me my storage accounts" \
+       --prompt "list storage accounts" \
+       --prompt "what storage do I have"
+     ```
+
+   - **Custom tools and prompts files** (use your own files for comprehensive testing):
+
+     ```bash
+     # Prompts:
+     # Use markdown format (same as e2eTests/e2eTestPrompts.md):
+     dotnet run -- --prompts-file my-prompts.md
+     
+     # Use JSON format:
+     dotnet run -- --prompts-file my-prompts.json
+     
+     # Tools:
+     # Use JSON format (same as eng/tools/PromptConfidenceScore/tools.json):
+     dotnet run -- --tools-file my-tools.json
+
+     # Combine both:
+     # Use custom tools and prompts files together:
+     dotnet run -- --tools-file my-tools.json --prompts-file my-prompts.md
+     ```
+
+   - **Quality assessment guidelines**:
+     - Aim for your description to rank in the top 3 results (GOOD or EXCELLENT rating)
+     - Test with multiple different prompts that users might use
+     - Consider common synonyms and alternative phrasings in your descriptions
+     - If validation shows POOR results, refine your description and test again
```

Comment:
Should POOR just fail validation? Seems like doing that would make this an easier gate.